scanner/phoenix.go

```diff
@@ -108,11 +108,16 @@ a Postgres database.
 `
 	}
 
-	// Add migration task if we find one of the dependencies that would run migrations.
-	// They are listed here: https://github.com/elixir-ecto/ecto?tab=readme-ov-file#usage
-	if checksPass(sourceDir, dirContains("mix.exs", "postgrex", "myxql", "tds")) {
+	if checksPass(sourceDir, dirContains("mix.exs", "postgrex")) {
+		s.DatabaseDesired = DatabaseKindPostgres
+		s.ReleaseCmd = "/app/bin/migrate"
+	} else if checksPass(sourceDir, dirContains("mix.exs", "myxql")) {
+		s.DatabaseDesired = DatabaseKindMySQL
+		s.ReleaseCmd = "/app/bin/migrate"
+	} else if checksPass(sourceDir, dirContains("mix.exs", "tds")) {
 		s.ReleaseCmd = "/app/bin/migrate"
```

Comment:
I'm not sure if we should set `s.ReleaseCmd` for MySQL and MSSQL databases, since we don't support them and the release command will fail. I just did this to keep the previous behavior, but I guess we should remove it?

Reply:
Yeah, if we don't support them then we probably shouldn't add handling for them.